client/src/elm/Main.elm

```diff
@@ -49,7 +44,7 @@ init flags =
         { room } =
             flags
     in
-        ( RoomModel [] room "This is a description" (Entry ""), getEntries room )
+        ( RoomModel [] room "This is a description" False (Entry ""), getEntries room )
```

Comment:
Tror kanskje begynner å bli bedre å bruke den andre formen for å initiere et objekt.